buildSrc/src/main/kotlin/dokka-conventions.gradle.kts

```diff
@@ -0,0 +1,82 @@
+import gradle.kotlin.dsl.accessors._74857f9dd946e843b9fa7c149d123d57.dokkatoo
```

Comment:
Redundant?

Reply:
yes, IntelliJ sometimes adds this when copy/pasting between build scripts.